sunpy/version.py

```diff
@@ -15,9 +15,3 @@
     del warnings
 
     version = '0.0.0'
-
-from packaging.version import parse as _parse
-
-_version = _parse(version)
-major, minor, bugfix = [*_version.release, 0][:3]
-release = not _version.is_devrelease
```

Comment:
I assume we are using this, but I wonder if we should add it to the template or not, given it introduces a packaging dependency.

Reply:
I do not think we are?